### DISCUSSION TITLE
1981-08-09-harold-fieldings-sunday-night-at-the-blackpool-opera-house/

### DISCUSSION THREAD

**Discussion by giscus** [FanzOfLenaZavaroni/fanzoflenazavaroni.github.io#47] [General]
# 1981-08-09-harold-fieldings-sunday-night-at-the-blackpool-opera-house/

Two shows at 6.10pm and 8.20pm plus a rehearsal at 2.30pm The Contract for the show was signed by Victor Zavaroni.

https://fanzoflenazavaroni.github.io/1981-08-09-harold-fieldings-sunday-night-at-the-blackpool-opera-house/

**Comment by StephenMolloy1** [comment]
I wonder if this was because Lena's parents lived in Rothesay and therefore her work permits were granted by Strathclyde council.  He might have had to take the contract to them so that they could approve it and issue a permit. Perhaps it was faster for him to sign it and forward it to Harold Fielding than to send it to Dorothy Solomon first who would then sign it and send it to Harold Fielding.

**Reply by FanzOfLenaZavaroni** [reply]
Good point! One of the many small details we may never learn about.
